### PR TITLE
feat: add windowsHide support

### PR DESCRIPTION
**Pull request by qingdengyue**
##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message follows commit guidelines

##### Description of change

1.chore: add node 12 support
2.fix: should send multi app worker test failed.
3.feature: add windowsHide support . 

close https://github.com/eggjs/egg/issues/3728

### CODE REVIEW

**Comment by qingdengyue** [comment]
cc @atian25 提交好了，看一下实现是否可行~ 文档还没添加，这个确定可以了，我再更新文档~  :smile:

**Comment by qingdengyue** [comment]
https://github.com/eggjs/egg/issues/3728

File: lib/master.js [hunk 1]
```diff
@@ -213,7 +214,9 @@ class Master extends EventEmitter {
     this.agentStartTime = Date.now();
 
     const args = [ JSON.stringify(this.options) ];
-    const opt = {};
+    const opt = {
+      windowsHide: this.options.windowsHide,
```
Comment (atian25) [inline lib/master.js]: 仅 win 的时候再加这个参数，也不需要外面传递了，直接写死 true

Comment (qingdengyue) [reply lib/master.js]: cfork 那边也改成了根据win直接设置为true

**Comment by atian25** [comment]
> codecov/patch — 50% of diff hit (target 98.39%)

这是没触发重跑么？@qingdengyue 本地跑跑 cov 看看？

**Comment by dead-horse** [comment]
加一个 windowsHide 的测试用例

**Comment by atian25** [comment]
> 加一个 windowsHide 的测试用例

怎么加？

**Comment by dead-horse** [comment]
> 怎么加？

看错了，现在是 windows 下直接就开启了

**Comment by dead-horse** [comment]
1.24.0

**Pull request merged**